docs/helpers/snippets/decodeJwt.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+import type { CredentialResponse } from "vue3-google-signin";
+import { GoogleSignIn } from "vue3-google-signin";
+import { decodeJWTToken } from "vue3-google-signin/utils";
```

Comment:
`import { decodeJWTToken } from "vue3-google-signin";` (its not coming from utils)

Reply:
A miss on my end. Do I need to import the function somewhere so it works that way ?